kotlinx-coroutines-core/jvm/test/guide/test/ExceptionsGuideTest.kt

```diff
@@ -9,7 +9,7 @@ class ExceptionsGuideTest {
     fun testExampleExceptions01() {
         test("ExampleExceptions01") { kotlinx.coroutines.guide.exampleExceptions01.main() }.verifyExceptions(
             "Throwing exception from launch",
-            "Exception in thread \"DefaultDispatcher-worker-2 @coroutine#2\" java.lang.IndexOutOfBoundsException",
+            "Exception in thread \"DefaultDispatcher-worker-1 @coroutine#2\" java.lang.IndexOutOfBoundsException",
```

Comment:
This is expected to vary from library version to library version (and, in more complex scenarios, from execution to execution) and isn't important: the number describes which of the many threads that were available managed to snatch the task and run it.

Reply:
I understand that this is a minor thing; just wanted to align outputs because I wasn't able to obtain another result, and I didn't want to make readers pay their attention to this difference. Can we leave it with my corrections, or should I revert them?

Reply:
If you feel strongly about it, we can keep it, but be aware that this number may change arbitrarily at any point.

Reply:
> If you feel strongly about it

No, I don't, but if it actually doesn't matter if there's 1 or 2 – I'd offer to merge this PR as is.